### ISSUE TITLE
Trim trailing subtitle whitespace before rendering

### ISSUE THREAD

**Issue by larsyencken** [owid/owid-grapher#1092]
## Problem

For ~15 charts we have subtitles with empty newlines, wasting whitespace that could be used for the visualisation.

The real problem is that FASTT like this was not trimmed at entry, most likely in the Grapher admin.

## Proposed solution

- [x] Adapt the grapher admin to trim whitespace on input for title, subtitle and other FASTT fields for _charts_ and _variables_
- [ ] Do a once-off migration that trims these fields, if possible (otherwise manually fix them)

Note: the fields we care about might be in a JSON blob, which could complicate the SQL we need.

## Examples

https://ourworldindata.org/grapher/per-capita-energy-use
https://ourworldindata.org/grapher/maternal-mortality-ratio-vs-gdp-per-capita

You can find more by querying for them:

```sql
select id, slug, subtitle
from (
  select
    id,
    json_unquote(json_extract(config, "$.slug")) slug,
    json_unquote(json_extract(config, "$.subtitle")) subtitle
  from charts
) c
where
  subtitle is not null
  and subtitle like "%\n"
```

**Comment by ShwetKhatri2001** [comment]
Hi @larsyencken !  I'd like to work on this issue.

**Comment by marcelgerber** [comment]
This is also related to owid/owid-issues#143, which is about cleaning this up at the database level.

**Comment by marcelgerber** [comment]
@ShwetKhatri2001 sure! please go ahead, and let us know if you need any help :)

**Comment by ikesau** [comment]
I've fixed this issue in the grapher editor with #1375 

@danyx23 tagging you to remind you to do the same for the bulk FASTT editor 🙂
